commercetools-models/src/main/java/io/sphere/sdk/carts/CustomLineItemDraft.java

```diff
@@ -27,19 +28,121 @@
 
     String getSlug();
 
+    /**
+     * Gets the tax rate for this custom line item.
```

Comment:
The tax category will be used to get the tax rate for this custom line item when the cart tax mode is External.